.github/workflows/ci-artifacts.yml

```diff
@@ -64,6 +59,24 @@ jobs:
         with:
           name: git-artifacts
           path: git-artifacts.tar.gz
+      - name: create zip and 7z SFX variants of the minimal SDK
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        shell: bash
+        run: |
+          for path in mingw64/bin/7z.exe mingw64/bin/7z.dll mingw64/lib/7zip/7zCon.sfx
+          do
+            git --git-dir=git-sdk-64.git show HEAD:$path >${path##*/}
+          done &&
+          mkdir minimal-sdk-extra &&
+          (cd minimal-sdk && ../7z.exe a -mx9 ../minimal-sdk-extra/git-sdk-x86_64-minimal.zip * .?*) &&
+          (cd minimal-sdk && ../7z.exe a -t7z -m0=lzma -mqs -mlc=8 -mx=9 -md=256M -mfb=273 -ms=256M -sfx../7zCon.sfx \
```

Comment:
IIRC (I believe I saw it some time last year when testing some pipelines on a self-hosted runner), 7-Zip can't use multiple cores for creating some types of archives. Not sure if that applies here as well, but I wonder if running these commands in parallel would speed up [the 2m+ runtime](https://github.com/dscho/git-sdk-64/actions/runs/10851317340/job/30114789179) here. GitHub recently [upgraded](https://github.blog/news-insights/product-news/github-hosted-runners-double-the-power-for-open-source/) OSS runners to have 4 cores, so that might help us here! WDYT?

Reply:
I did try to add `-mmt=on` (which in my local tests made the `.zip` packing twice as fast). However, both [without](https://github.com/dscho/git-sdk-64/actions/runs/10851317340/job/30114789179#step:11:1) as well as [with](https://github.com/dscho/git-sdk-64/actions/runs/10856380398/job/30130917458#step:11:1) multi-threading, it takes 2 minutes. So I assume that it does not really multi-thread...

Now, we _could_ upload an artifact containing the `7z.exe`, `7.zdll` and `7zCon.sfx` files, then add another job that does the `.zip`/`.7z.exe` packing, and make that another dependency of the `publish-release-assets`  job, but that's just complicating things, and since this workflow is supposed to run at most once a day, I am willing to just be okay with the added 2 minutes. Sound good?

Reply:
> I am willing to just be okay with the added 2 minutes. Sound good?

Yeah, let's leave it as is then. Thanks!